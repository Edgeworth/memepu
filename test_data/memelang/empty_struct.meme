// status: ok
// input: ''
// output: ''
// returns: 0
struct Empty {}
fn main() i8 { ret 0; }
