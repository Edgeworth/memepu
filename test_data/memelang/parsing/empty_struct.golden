Module
 File(empty_struct.meme)
  Fn(main)
   FnSig
    Typename(main)
    Type(Ref(i8))
     Ref(i8)
   StmtBlk
    Ret
     IntLit(0)
  Struct
   Typename(Empty)
